src/ui/utils/apolloClient.tsx

```diff
@@ -181,9 +183,15 @@ function createHttpLink(token: string | undefined) {
 }
 
 function createErrorLink(onAuthError?: () => void) {
-  return onError(({ graphQLErrors, networkError }) => {
+  return onError(err => {
+    const { graphQLErrors, networkError } = err;
     if (graphQLErrors) {
-      console.error("[Apollo GraphQL error]", graphQLErrors);
+      console.error(
+        "[Apollo GraphQL error]",
+        err.operation.operationName,
+        extractGraphQLError(err),
+        graphQLErrors.length > 1 ? graphQLErrors : graphQLErrors[0]
```

Comment:
I'm a little confused by this. So we log

```
[error]
```
if there is only one, or
```
error
```
if there is more than one? Since this is just a `console.error` call I'm sure it's fine either way, but just wondering if I'm missing something meaningful here.

Reply:
There's usually only one error so if that's the case I'm pulling that object out so sentry doesn't try to be too smart and collapse the output to "Array" because it's a deep object. But, if there are multiple errors I'm not trying to be too fancy and just logging them all.

Reply:
Gotchya, I still wish we had something more specific at the top-level other than `[Apollo GraphQL error]`, just for Sentry's grouping reasons, but this seems better than silencing everything.